contrib/packaging_python/mk_clean_wheel.sh

```diff
@@ -44,7 +45,7 @@ fi
 # configure cvc5
 echo "Configuring"
 rm -rf build_wheel/
-python contrib/packaging_python/mk_build_dir.py $CONFIG --python-bindings --name=build_wheel
+./configure.sh $CONFIG --python-bindings --name=build_wheel -DPython_FIND_VIRTUALENV=ONLY -DPYTHON_LIBRARY=$VIRTUAL_ENV/lib -DPYTHON_INCLUDE_DIR=$PYINCLUDE
```

Comment:
This new command like is supposed to ensure that we use the python version from the virtual environment that is activated. Once we remove scikit, this will not be necessary.

Reply:
Again, I think you can add the above comment to the script.